rclpy/rclpy/parameter.py

```diff
@@ -23,7 +23,7 @@
 
 class Parameter:
 
-    class Type(Enum):
+    class Type(IntEnum):
```

Comment:
Using IntEnum as a base class will change the behavior, which will make compareing a variable of Type with int value return True. Maybe the author wants the variable explicitly compare with the Type value, such as Type.BOOL, not int(1).

What about updating from
```
response.types.append(param.type_)
```
to
```
response.types.append(param.value_)
```

Reply:
i see your point, thanks. let's keep this open to get more feedback.

Reply:
Maybe we can just update "_type"  by appending ".value"

Reply:
@iuhilnehc-ynos alternative also sounds reasonable.

Reply:
@fujitatomoya can you apply @iuhilnehc-ynos suggestion, instead of modifying `Enum` to `IntEnum`.
I think that not modifying the type is a better idea.

Reply:
yeah, okay i will do that in today.

Reply:
> Maybe we can just update "type_" by appending ".value" for

thanks for the suggestion, it's been addressed with this one.